recipe/meta.yaml

```diff
@@ -10,43 +10,60 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
+  patches:
+    - 0001-remove-obsolete-components.patch
 
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
-  noarch: python
 
 requirements:
   build:
+    - patch  # [not win]
+    - m2-patch  # [win]
+  host:
     - python
     - setuptools
     - pip
+    - wheel
   run:
     - python
-    - pyyaml >=5.1.1
-    - flask >=1.0.2
+    - flask >=1.0.4
+    - plotly >=5.0.0
+    - importlib-metadata ==4.8.3  # [py<37]
+    - contextvars ==2.4  # [py<37]
+    # compress extra
     - flask-compress
```

Comment:
`flask-compress` is an `extras_require` dependency https://github.com/plotly/dash/blob/680a9b05a14ecaac15cf09d3f8838d3750ed7915/setup.py#L36. Should it be inside `run_constrained`?

Reply:
It is an extra now. Since this was included in prior package builds, I didn't want to remove it unless there was a compelling reason for it.